ext/uri/php_uri.c

```diff
@@ -354,6 +371,7 @@ ZEND_ATTRIBUTE_NONNULL_ARGS(1, 2) PHPAPI void php_uri_instantiate_uri(
 			if (pass_errors_by_ref_and_free(errors_zv, &errors) == FAILURE) {
 				RETURN_THROWS();
 			}
+			zval_ptr_dtor(return_value);
```

Comment:
Why is this line necessary? Also makes me wonder if the RETURN_NULL() line violates any stub...